### ISSUE TITLE
`lando init` fails with: "error: Lando thinks you might need some help with your droid"

### ISSUE THREAD

**Issue by benjaminbradley** [lando/lando#1158]
Bug Report
==========
Lando version: lando 3.0.0-beta.47 (latest release at time of writing)
OS: MacOS High Sierra 10.13.6 (17G65)

STEPS TO REPRODUCE:
1. cd into a directory of a git repository for a pantheon project
2. run `lando init`

ACTUAL: RESULTS:
> ? What recipe do you want to use? pantheon
> ? Choose a Pantheon account [my email]
> ? Which site? [my project]
> error: Lando thinks you might need some help with your droid

This error message provides no useful information to help troubleshoot the issue.

I tried `lando init -vvvv` and got no additional info.

I would suggest improvement of the error message to help the user identify and resolve whatever went wrong. Googling the phrase above only returns search results about how Lando Calrissian is pansexual in the latest Star Wars films.

**Comment by pirog** [comment]
@benjaminbradley this is an impressive error to get, means you haven't installed Docker. How are you installing?

**Comment by benjaminbradley** [comment]
@pirog I have docker installed and have been using it for a couple years. I want to install lando for the pantheon-sync support. When I ran the Lando installer, the Docker part seemed to be optional so I un-checked it. Do I need to set some environment variables to get lando to see my docker install?
The error message should communicate that it can't find docker, and ideally offer some guidance as to how to resolve the issue. The funny message is fine to include (and would be much more appreciated) as long as the helpful info is also present.

**Comment by pirog** [comment]
Yeah I mean the funny message is there because it should be close to
impossible to actually get that error. You are literally the first in like
20,000 people to actually get it so congratulations

**Comment by benjaminbradley** [comment]
@pirog ok I updated to the latest version of Docker (18.06.1-ce-mac73, previous version was 18.06.1-ce-mac70), and I'm still getting the same error. docker is present in the path.

What's your advice to troubleshoot this issue?

**Comment by andymelt** [comment]
I'm getting the same error. On Docker version 18.06.1-ce-mac73 (26764). macOS High Sierra 10.13.6 (17G65). I've never seen this before. Very strange.

**Comment by andymelt** [comment]
After digging around, I found that I had indeed (inadvertently) moved the Docker.app file. Moved it back into /Applications where it was before and Lando is happy and doesn't make wisecracks about droids. :)

**Comment by benjaminbradley** [comment]
oh look at that! I symlinked /Applications/Docker.app to where my Docker.app actually lives and lando is now COOKING WITH FIRE! yay.

**Comment by greggmarshall** [comment]
I get this error on a Windows 10 machine.  When I check Docker from the command line, I can see it is fine, albeit possibly old.  I agree it is an error message with no useful information.

**Comment by Idealien** [comment]
Also getting this error on Windows 10 system. Installed Docker Toolbox (Docker version 18.03.0-ce, build 0520e24302) separate to Lando installation. I need to stick with Virtualbox for short-term which prevents Hyper-V + Docker for Windows.

**Comment by pirog** [comment]
We should probably augment the error message to include a link to the install instructions. If you do not install Docker via Lando then you need to install it in the default way otherwise things will likely not work.

**Comment by idpokute** [comment]
I am using docker-toolbox on windows 10 home edition, and I met the same issue. @Idealien Hows it going so far? I am trying to downgrade lando version, but so far no luck.

**Comment by Idealien** [comment]
The Hyper-V requirement is blocker to me still. I have moved on to [Minishift ](https://blog.openshift.com/running-wordpress-easy-way/) for local dev which provides better dev/prod parity for it's use of Kubernetes.

**Comment by pirog** [comment]
Lando only officially supports using Docker for Windows with the factory default settings. While it probably can be made to work in other configurations you are definitely on your own if you go that route

**Comment by grantkruger** [comment]
I get this error while trying to uninstall Lando. I did first uninstall Docker, so now it seems I have to reinstall docker so that I can uninstall it again with Lando.

**Comment by pirog** [comment]
in RC2 we've improved the error message here so that users can verify they have followed the proper installation instructions
